test/deprecated/targetLocales.good

```diff
@@ -0,0 +1,6 @@
+targetLocales.chpl:9: warning: 'targetLocales' method on arrays, domains, and distributions is now paren-less; please invoke it without parentheses
```

Comment:
Does this test need an alternate .good file for gasnet?

Reply:
Turns out it doesn't. Tests without .numlocales should in general behave the same with --local and with gasnet because paratest runs them on a single locale.

I double-checked this test under gasnet nonetheless. It passes.